### DISCUSSION TITLE
[Tabs] Close current active Tab content when clicking the trigger again

### DISCUSSION THREAD

**Discussion by franciscohanna92** [radix-ui/primitives#2548] [Help]
I'm using Tabs and I need to close all tabs when clicking the active one again. 

Example: Clicking the "Reuse" tab trigger, it's content should be closed.
![image](https://github.com/radix-ui/primitives/assets/12707758/57a95aed-649b-4a4c-b9ad-b67024eb089e)

Right now, the workaround is to use either `onMouseDown` or `onPointerDown` but this feels hacky. Is there any other way to do this?

**Comment by benoitgrelard** [comment]
That functionality isn't really a common thing with application tabs so we won't support it.
